src/main/java/org/folio/marccat/integration/TenantRefService.java

```diff
@@ -34,36 +35,36 @@
   @Autowired
   private TenantLoading tl;
 
-
   /**
    * Load data.
    *
-   * @param tenantAttributes the tenant attributes
-   * @param headers the headers
+   * @param tenantAttributes        the tenant attributes
+   * @param headers                 the headers
+   * @param loadBibliographicSample
    */
-  public void loadData(TenantAttributes tenantAttributes, Map<String, String> headers){
+  public void loadData(TenantAttributes tenantAttributes, Map<String, String> headers) {
     logger.debug("Start sample data loading");
     boolean loadData = buildDataLoadingParameters(tenantAttributes, tl);
     logger.debug("Is Load data " + loadData);
-  // if (loadData) {
-      tl.perform(tenantAttributes, headers);
-   // }
+    if (loadData) {
```

Comment:
As you can see in the TenantRefService class,
the boolean loadData in the loadData (...) method is commented out because
together with Mirko we have that loadSample parameter was always false under Okapi.
We did several tests a while ago. We have also updated the _tenant interface v1.2 (ModuleDescriptor-template.json).

Reply:
Ok, then I have commented out the condition again